docs/streams/developer-guide/app-reset-tool.html

```diff
@@ -106,6 +107,11 @@ <h2>Step 1: Run the application reset tool<a class="headerlink" href="#step-1-ru
                                         topics <span class="o">(</span>topics used in the through<span class="o">()</span>
                                         method<span class="o">)</span>. For these topics, the tool
                                         will skip to the end.
+--internal-topics &lt;String: list&gt;      Comma-separated list of internal topics
+                                        to delete. Must be a subset of the
+                                        internal topics marked for deletion by
+                                        the default behaviour (do a dry-run without
+                                        this option to view these topics).
```

Comment:
screenshot of this portion:

<img width="1031" alt="Screenshot 2020-08-13 at 2 19 35 PM" src="https://user-images.githubusercontent.com/32009741/90100950-52982600-dd70-11ea-9ae5-2b84296188cc.png">